ferry.go

```diff
@@ -38,6 +38,13 @@ const (
 	StateDone                = "done"
 )
 
+func quoteFields(fields []string) (out []string) {
```

Comment:
Minor nit: might want to have this in `utils.go` (along with `quoteField`) as I can see here.